themes/classic/templates/catalog/_partials/product-details.tpl

```diff
@@ -50,7 +50,11 @@
           <dl class="data-sheet">
             {foreach from=$product.features item=feature}
               <dt class="name">{$feature.name}</dt>
-              <dd class="value">{$feature.value}</dd>
+              {if $feature.id_feature === '4'}
```

Comment:
Yes, it does, it's important to remember that features are only for information and shouldn't be dynamic, unless we'll have the feature to... have features related to combinations :)